Meta/build-root-filesystem.sh

```diff
@@ -158,7 +158,11 @@ chown -R 100:100 mnt/home/anon/Desktop
 echo "done"
 
 printf "installing shortcuts... "
-ln -sf Shell mnt/bin/sh
+if [ -e mnt/usr/local/bin/bash ]; then
+    ln -sf bash mnt/bin/sh
+else
+    ln -sf Shell mnt/bin/sh
+fi
```

Comment:
I don't know about changing the system shell if a port is installed or not. Does `pkgsrc` expect `/bin/sh` to be POSIX-compatible? Can that be changed on the `pkgsrc` side instead?

Reply:
This wouldn't be necessary if `/bin/Shell` was POSIX-compliant... Until then, this is a compromise where the login shell of users is still `/bin/Shell`, but `/bin/sh` becomes POSIX-compliant when the bash port is installed.

Reply:
If you want to do this sort of hack, please do it locally on your machine.